docs/pages/Troubleshooting.md

```diff
@@ -9,6 +9,37 @@ You need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://n
 ## Issues Not Listed Here
 Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#report-an-issue) on how to report an issue.
 
+## UI5 Project
+### Corrupt Framework Dependencies
+
+When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.
+
+These issues can be the result of an aborted install during a preceding UI5 Tooling execution. Because of [a bug](https://github.com/SAP/ui5-tooling/issues/478), that has since been fixed, UI5 Tooling assumed that the preceding install was successful and work with the potentially corrupted dependency.
```

Comment:
```suggestion
These issues can be the result of an aborted install during a preceding UI5 Tooling execution. Because of [a bug](https://github.com/SAP/ui5-tooling/issues/478) that has since been fixed, UI5 Tooling assumes that the preceding install was successful and uses the potentially corrupted dependency.
```